Practica real con tabla/ViewController.swift

```diff
@@ -10,16 +10,26 @@ import UIKit
 
 class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource  {
 
-
+    // data structure of message
+    class Message {
```

Comment:
Creates a new data type for saving messages.

Saves the content (a string), and a flag indicates whether the message is sent by others or not.